server/sensorAlerts.js

```diff
@@ -88,7 +114,62 @@ async function handleAlert(location, alertType) {
   }
 }
 
+const validateSensorEvent = Validator.body(['coreid', 'event', 'api_key', 'data']).exists()
```

Comment:
I see you moved all this code and added `data`. Did anything else change from when it was in `particle.js` to here?

And in those Sensors that are so old that the `data` is unusable, will they still pass something that the validator will consider as existing?